### PR TITLE
update to IA ideas

### PR DESCRIPTION
**Pull request by zekiel**
//cc @moollaza 

deleting my other PR

### CODE REVIEW

File: duckduckhack/getting-started/ddh-intro.md [hunk 1]
```diff
@@ -1,32 +1,110 @@
...
+## How can I contribute? 
+
+Create new Instant Answers.  
+*Use this documentation to develop a new Instant Answer*  
+
+Improve Current Instant Answers.  
+*Fix issues in our [zeroclickinfo repositories](https://github.com/duckduckgo)*  
+
+Code review and help  
+*Help with pull requests in our [zeroclickinfo repositories](https://github.com/duckduckgo))*
```
Comment (moollaza) [inline duckduckhack/getting-started/ddh-intro.md]: double `)` at the end

File: duckduckhack/getting-started/ddh-intro.md [hunk 2]
```diff
@@ -1,32 +1,110 @@
...
+
+In both of these examples, users will more easily find what they're looking for via the Instant Answers provided, instead of clicking through regular results. If you're still unsure of what to build, you can work on an idea from our community or get help with your idea here: https://duck.co/ideas 
+
+#### Ideas to avoid
+
+The DuckDuckGo audience is diverse in age and location. Because of our audience and some restrictions with licensing, there are a few types of Instant Answers we cannot accept at this time (though, it's always best to ask if you're not sure). Currently, the list of restricted Instant Answers includes: 
+
+
+**Content prohibited**  
+-Adult content
```
Comment (moollaza) [inline duckduckhack/getting-started/ddh-intro.md]: @zekiel did you want these to display as bulletted items? they'll need a space after the `-` and the following lines will need to be indented -- I can do this super easily

Comment (zekiel) [inline duckduckhack/getting-started/ddh-intro.md]: i thought it might be a bit visually dominating but maybe for consistency.

**Comment by zekiel** [comment]
merging 

//cc @talsraviv  --- I'll be looking at your other PR in a moment. This PR here is a precursor to the future Welcome update we talked about.

**Pull request merged**